scalability/statemate.c

```diff
@@ -267,25 +267,25 @@ void init(void)
-   klee_make_symbolic(&NICHT_INITIALISIERT_NICHT_INITIALISIERT_next_state, sizeof(NICHT_INITIALISIERT_NICHT_INITIALISIERT_next_state), "NICHT_INITIALISIERT_NICHT_INITIALISIERT_next_state");
-   klee_make_symbolic(&ZENTRAL_KINDERSICHERUNG_CTRL_next_state, sizeof(ZENTRAL_KINDERSICHERUNG_CTRL_next_state), "ZENTRAL_KINDERSICHERUNG_CTRL_next_state");
-   klee_make_symbolic(&MEC_KINDERSICHERUNG_CTRL_next_state, sizeof(MEC_KINDERSICHERUNG_CTRL_next_state), "MEC_KINDERSICHERUNG_CTRL_next_state");
-   klee_make_symbolic(&KINDERSICHERUNG_CTRL_KINDERSICHERUNG_CTRL_next_state, sizeof(KINDERSICHERUNG_CTRL_KINDERSICHERUNG_CTRL_next_state), "KINDERSICHERUNG_CTRL_KINDERSICHERUNG_CTRL_next_state");
-   klee_make_symbolic(&B_FH_TUERMODUL_CTRL_next_state, sizeof(B_FH_TUERMODUL_CTRL_next_state), "B_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&A_FH_TUERMODUL_CTRL_next_state, sizeof(A_FH_TUERMODUL_CTRL_next_state), "A_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&WIEDERHOLSPERRE_FH_TUERMODUL_CTRL_next_state, sizeof(WIEDERHOLSPERRE_FH_TUERMODUL_CTRL_next_state), "WIEDERHOLSPERRE_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&INITIALISIERT_FH_TUERMODUL_CTRL_next_state, sizeof(INITIALISIERT_FH_TUERMODUL_CTRL_next_state), "INITIALISIERT_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&TIPP_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state, sizeof(TIPP_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state), "TIPP_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&MANUELL_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state, sizeof(MANUELL_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state), "MANUELL_SCHLIESSEN_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&OEFFNEN_FH_TUERMODUL_CTRL_next_state, sizeof(OEFFNEN_FH_TUERMODUL_CTRL_next_state), "OEFFNEN_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&SCHLIESSEN_FH_TUERMODUL_CTRL_next_state, sizeof(SCHLIESSEN_FH_TUERMODUL_CTRL_next_state), "SCHLIESSEN_FH_TUERMODUL_CTRL_next_state");
-   klee_make_symbolic(&FH_STEUERUNG_DUMMY_FH_STEUERUNG_DUMMY_next_state, sizeof(FH_STEUERUNG_DUMMY_FH_STEUERUNG_DUMMY_next_state), "FH_STEUERUNG_DUMMY_FH_STEUERUNG_DUMMY_next_state");
-   klee_make_symbolic(&EINKLEMMSCHUTZ_CTRL_EINKLEMMSCHUTZ_CTRL_next_state, sizeof(EINKLEMMSCHUTZ_CTRL_EINKLEMMSCHUTZ_CTRL_next_state), "EINKLEMMSCHUTZ_CTRL_EINKLEMMSCHUTZ_CTRL_next_state");
-   klee_make_symbolic(&BEWEGUNG_BLOCK_ERKENNUNG_CTRL_next_state, sizeof(BEWEGUNG_BLOCK_ERKENNUNG_CTRL_next_state), "BEWEGUNG_BLOCK_ERKENNUNG_CTRL_next_state");
-   klee_make_symbolic(&BLOCK_ERKENNUNG_CTRL_BLOCK_ERKENNUNG_CTRL_next_state, sizeof(BLOCK_ERKENNUNG_CTRL_BLOCK_ERKENNUNG_CTRL_next_state), "BLOCK_ERKENNUNG_CTRL_BLOCK_ERKENNUNG_CTRL_next_state");
+/*   klee_make_symbolic(&tm_entered_EINSCHALTSTROM_MESSEN_BLOCK_ERKENNUNG_CTRLch_BLOCK_ERKENNUNG_CTRL__N_copy, sizeof(tm_entered_EINSCHALTSTROM_MESSEN_BLOCK_ERKENNUNG_CTRLch_BLOCK_ERKENNUNG_CTRL__N_copy), "tm_entered_EINSCHALTSTROM_MESSEN_BLOCK_ERKENNUNG_CTRLch_BLOCK_ERKENNUNG_CTRL__N_copy"); */
```

Comment:
@rasoolmaghareh Different from `nsichneu.c`, having comments here would be good, but perhaps adding some comment on why some variables such as `tm_entered_EINSCHALTSTROM_MESSEN_BLOCK_ERKENNUNG_CTRLch_BLOCK_ERKENNUNG_CTRL__N_copy` are not made symbolic would be good, just for the record.